frontend/src/scenes/events/definitions/definitionDrawerLogic.ts

```diff
@@ -121,6 +167,7 @@ export const definitionDrawerLogic = kea<definitionDrawerLogicType<EventOrPropTy
                 errorToast('Oops! This tag is already set', 'This event already includes the proposed tag.')
                 return
             }
+            actions.setType('event_definitions')
```

Comment:
I see a lot of `setType` action calls. This feels like a code smell somehow. If it's set in `openDrawer`, why should it ever change? If it's needed to make the right API calls, the `type` should be inside the action payload, not a "magic global".

Reply:
ok! yeah that was what I was thinking about, passing the url as an argument vs creating it as a global, I wasn't sure which was better practice

Reply:
but I leaned towards making it a global because it felt more kea aligned? haha idk, just my thought process

Reply:
The trouble with globals is that you can never be sure what the value is. You might set it to one type, but some other action elsewhere might reset it again... and when you read it and expect it to be one thing, it might be something else. End result: updating the wrong type of property.

Mostly things will be fine, but if we can get by with less moving parts, we should. Less parts = less code = simpler app = less bugs.